tests/helper/DynamoDBHelper.ts

```diff
@@ -0,0 +1,140 @@
+        ReadCapacityUnits: readCapacityUnits,
+        WriteCapacityUnits: writeCapacityUnits
+      },
+      SSESpecification: {
+        Enabled: true
+      }
+    };
+    return this.createTable(params, this.deploymentTableName);
+  }
+
+  async deleteDeploymentTable(): Promise<void> {
+    return await this.deleteTable(this.deploymentTableName);
+  }
+
+  async getEnv(teamName: string): Promise<AWS.DynamoDB.DocumentClient.AttributeMap> {
+    const params: AWS.DynamoDB.Types.DocumentClient.QueryInput = {
+      KeyConditionExpression: '#PK = :PK',
+      ExpressionAttributeNames: {
+        '#PK': 'PK'
+      },
+      ExpressionAttributeValues: {
+        ':PK': teamName
+      },
+      TableName: this.deploymentTableName,
+      ScanIndexForward: false,
+      ConsistentRead: true
+    };
+    return await this.get(params);
+  }
+
+  async getDeployment(teamName: string, environmentName: string): Promise<AWS.DynamoDB.DocumentClient.AttributeMap> {
+    const teamNameEnv = `${teamName}_${environmentName}`;
+    const params: AWS.DynamoDB.Types.DocumentClient.QueryInput = {
+      KeyConditionExpression: '#PK = :PK',
+      ExpressionAttributeNames: {
+        '#PK': 'PK'
+      },
+      ExpressionAttributeValues: {
+        ':PK': teamNameEnv
+      },
+      TableName: this.deploymentTableName,
+      ScanIndexForward: false,
+      ConsistentRead: true
+    };
+    return await this.get(params);
+  }
+
+  async addDeployment(deployment: Deployment): Promise<void> {
+    await this.add({
+      TableName: this.deploymentTableName,
+      Item: {
+        PK: deployment.teamName + '_' + deployment.environment,
+        SK: deployment.timestamp,
+        deployment: deployment,
+      }
+    });
+    await this.add({
+      TableName: this.deploymentTableName,
+      Item: {
+        PK: deployment.teamName,
+        SK: deployment.environment,
+      }
+    });
+  }
+
+  async countAllDeployments(): Promise<number> {
+    const params: DocumentClient.ScanInput = {
+      Select: 'COUNT',
+      TableName: this.deploymentTableName
+    };
+    const data: AWS.DynamoDB.DocumentClient.ScanOutput = await this.databaseClient.scan(params).promise();
+    return data.Count;
+  }
+
+  private async add(params: DocumentClient.PutItemInput): Promise<void> {
+    await this.databaseClient.put(params).promise();
+  }
+
+  private async get(params: DocumentClient.QueryInput): Promise<AWS.DynamoDB.DocumentClient.AttributeMap> {
+    const data: AWS.DynamoDB.DocumentClient.QueryOutput = await this.databaseClient.query(params).promise();
+    return data.Items;
+  }
+
+  private async createTable(params, tableName: string): Promise<void> {
+    await this.dynamoDb.createTable(params).promise();
+  }
+
+  private async deleteTable(tableName: string): Promise<void> {
+    try {
+      await this.describeTable(tableName);
+      await this.dynamoDb.deleteTable({TableName: tableName}).promise();
+    } catch (err) {
+      return null;
```

Comment:
IMO makes sense to log the error. Otherwise the error is just silently swallowed